lib/AuthTokenVerifier.ts

```diff
@@ -72,7 +78,7 @@ export class AuthTokenVerifier {
           const publickey = parts.length == 2 ? parts[1] : parts[0];
           const decodedClaim = await this._jwt.verify(token, publickey);
 
-          return decodedClaim !== undefined;
+          return (decodedClaim !== undefined || this.p256verify(token, publickey));
```

Comment:
PR https://github.com/energywebfoundation/ew-did-registry/pull/349 includes verification of ES256. To enable it, `jwt.verify` should be passed array of methods [`ES256`, `ES256K`]